packaging/linux/arch/update_aur_packages.sh

```diff
@@ -4,62 +4,19 @@
 
 set -e -u -o pipefail
 
-# build_binaries.sh and deb/package_binaries.sh must already have been run on
-# this build root.
-build_root="$1"
 here="$(dirname "$BASH_SOURCE")"
+. "$here/arch_common.sh"
```

Comment:
Nit: I like to use `source` instead of `.`, just to make it visually clear that sourcing is happening.